DemoApp-Swift/DemoApp-Swift/Ads/ALAutoLayoutBannerAdViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ALAutoLayoutBannerAdViewController.swift
+//  DemoApp-Swift
+//
+//  Created by Andrew Tian on 9/20/19.
+//  Copyright © 2019 AppLovin. All rights reserved.
+//
+
+import UIKit
+import AppLovinSDK
+
+class ALAutoLayoutBannerAdViewController: UIViewController, MAAdViewAdDelegate
+{
+    private var adView: MAAdView?
```

Comment:
Should be explicit, we know it'll always exist. If not, I would've [marked](https://github.com/AppLovin/AppLovin-MAX-SDK-iOS/blob/master/DemoApp-ObjC/DemoApp-ObjC/Ads/ALAutoLayoutBannerAdViewController.m#L13) it as `nullable` in Obj-C:

Anyways, the best way to do it if you can, is to in-line it:

```suggestion
    private let adView = MAAdView(adUnitIdentifier: "BANNER_AD_UNIT_ID")
```

You gain the benefit of `let` and it always being immutable as well. .